annotation-web/src/main/resources/annotation-web-context.xml

```diff
@@ -42,6 +42,15 @@
 	<beans:bean id="annotationService" class="eu.europeana.annotation.web.service.impl.AnnotationServiceImpl">
     </beans:bean>
     
+    <beans:bean id="subtitleHandler" class="eu.europeana.annotation.fulltext.subtitles.SubtitleHandler">
+    	<beans:constructor-arg index="0" value="#{annotationProperties['annotation.subtitles.formats.xml']}"/>
+    	<beans:constructor-arg index="1" ref="xmlJacksonMapper"/>
+    </beans:bean>
+    
+    <beans:bean id="xmlJacksonMapper" class="com.fasterxml.jackson.dataformat.xml.XmlMapper">
```

Comment:
this is not needed as a bean, as this mapper is used only for application initialization, it must not be used afterwards

Reply:
It may be used somewhere else, having that as a bean can only help. It is instantiated once anyway.